snake/common/variants/variants_snake.py

```diff
@@ -1,3 +1,70 @@
+# Freebayes calls a variant multiple times, when a bed file contains overlapping regions.
+# This rule uses bedtools merge to flatten a bedfile.
+
+rule mergeRegionsInBed:
+    input:
+        regions = config['resources'][ORGANISM]['regions']
+    output:
+        flat_bed = FREEBAYESOUT + '_mergedRegions.bed'
```

Comment:
I thing it should be 
flat_bed = config['resources'][ORGANISM]['regions'] + '_mergedRegions.bed'
. Otherwise it is only usable for freebayes and not in general